design/SAS/sas_weighted_mmrm.sas

```diff
@@ -0,0 +1,28 @@
+* Weighted MMRM ----;
+
+** ML ----;
+
+ODS RTF FILE="&datpath\sas_weighted_mmrm_ml.rtf";
```

Comment:
thanks @ywang-gilead , one minor ask, can you please save the SAS results as txt files? because then it is much easier to view them during code review etc. thank you!

Reply:
Thanks for considering the implementation of the weight statement in PROC MIXED! Just as a remark, not sure it should become an issue, at least not a pressing one: I was also thinking about use cases for weights that are sampling weights, more in the sense of non-random survey samples, to model e.g. channeling into treatments when using RWD. This is implemented in SAS survey procedures, e.g. PROC SURVEYREG, and also in PROC GLIMMIX ("obsweight" in MODEL statement and "weight" in RANDOM statement, to weight observations and subjects (higher levels), respectively). 
See e.g. https://documentation.sas.com/doc/en/pgmsascdc/9.4_3.3/statug/statug_glimmix_details25.htm . Of note perhaps, PROC GLIMMIX does this with G-side random effects only, not R-side random effects.

Reply:
Thanks @chstock - let me move this to a separate issue and then we can discuss further there / in a dedicated meeting.